textstyles/texts.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="texts.css">
+</head>
+<body>
+    <div class="text-utilities">
+        <div class="h1">
```

Comment:
I don't think you need both div and h tags. You can use either 

<div class="h1">Heading 1</div>

or reset the h1 tag according to your need and just use
<h1>Heading 1</h1>

Same for all the other H tags